k8s/api-deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose convert
+    kompose.version: 1.17.0 (HEAD)
+  labels:
+    io.kompose.service: api
+  name: api
+spec:
+  replicas: 1
+  selector:
+    matchLabels: 
+      io.kompose.service: api
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        io.kompose.service: api
+    spec:
+      containers:
+      - args:
+        - uwsgi
+        - --uid
+        - guest
+        - --gid
+        - nogroup
+        - --http
+        - :9876
+        - --socket
+        - :9877
+        - --manage-script-name
+        - --mount
+        - /app=monocle.webapp:app
+        env:
```

Comment:
Would it be better to set the envs via a  configMap ?

Reply:
Yeah, the implementation of this was quite naive, changes can be made to separate concerns and make it more readable